index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
 
```

Comment:
```suggestion
give more instructions on what other actions can be used
```

Reply:
This is a "suggestion" but not a "suggested change": a suggested change is where you actually modify it to make it the way you want it to be. Your comment is totally fine, but I would submit that as a *comment* rather than a *suggested change*.